README.md

```diff
@@ -79,130 +87,17 @@ Add Alby to your browser
 
 ## 🚀 Quick Start
 
-Ensure you have
-
-- [Node.js](https://nodejs.org) v16 or newer installed (we run tests with v18)
-- [Yarn](https://yarnpkg.com) v1 or v2 installed
-
-Supported but not required
-
-- [nvm](https://github.com/nvm-sh/nvm#intro)
-
-Then run the following
-
-1. Install dependencies\
-   `yarn install`
-1. Start the development server for the extension
-   - `yarn run dev:chrome`
-   - `yarn run dev:firefox`
-   - `yarn run dev:opera`
-1. To build the extension
-   - `yarn run build:chrome`
-   - `yarn run build:firefox`
-   - `yarn run build:opera`
-1. Build and pack extensions all at once to the `dist/production` directory\
-   `yarn run build`
-1. Build the production packages in the `dist/production` directory\
-   `yarn run package`
-
-### 🛠 Development
-
-- Install dependencies\
-  `yarn install`
-- To watch file changes in development
-  - Chrome\
-    `yarn run dev:chrome`
-  - Firefox\
-    `yarn run dev:firefox`
-  - Opera\
-    `yarn run dev:opera`
-
-#### Testnet/testing-accounts for development
-
 We set up our own internal testnet, which can be used for your development.
 If this is not reachable please let us know.
 
 - [Test-setup](https://github.com/getAlby/lightning-browser-extension/wiki/Test-setup) for different connectors (i.e. LND)
-- [RTL](https://rtl.regtest.getalby.com) for testing nodes (PW: `getalby`)
+- [Thunderhub](https://thunderhub.regtest.getalby.com/) for testing nodes (PW: `getalby`)
   Currently only lists LND nodes
 - [LNDhub.go API Swagger](https://lndhub.regtest.getalby.com/swagger/index.html)
 
```

Comment:
Instead of repeating these lines again in setup.md, maybe we can link setup.md directly here itself saying "Refer to SETUP.md for info regarding how to setup Alby"? //cc @escapedcat

Reply:
Our wallet does not have the function of switching the main network and testing the network as metamask does, so I put this section on the home page. It is easier for new users to learn about the wallet through testing the network. This new user may not be a developer.
Maybe this text description is not suitable for new users to link to the test network. If there is a suitable text description, it would be better to put it here

Reply:
Let me modify it first to see the effect